### PR TITLE
Homework assignment - Events Lab

### PR DESCRIPTION
**Pull request by mradambeck**
Was able to do Change, Ready (within the console/dev tools, I believe that's what it was asking), Click, and Keypress.  Didn't get a chance to work on Submit.  Found formatting the time to actually be one of the more interesting challenges.

### CODE REVIEW

**Comment by mradambeck** [comment]
Updated, realized it was only working if I clicked.

File: change/base.js [hunk 1]
```diff
@@ -2,6 +2,20 @@ console.log("Sanity Check: JS is working!");
...
 $(document).ready(function(){
 
-  $("#time").text( Date.now() );
+  $("#time").text( Date.now() + " yr mom" );
 
-})
+  // sets all values to 0
+  $("input").val(0);
+  
+  $("input").on("change", function doMath(l, r){
```
Comment (Nick-Brennan) [inline change/base.js:10]: What is passed to the callback is predetermined by the function that calls it (in this case 'on').  doMath(event) or doMath(e) would make more sense

File: keypress/base.js [hunk 2]
```diff
@@ -2,4 +2,48 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-})
+  var $keyClick = false;
+  var $startTime = 0;
+  var $endTime = 0;
+  var $totalTime = 0;
+
```
Comment (Nick-Brennan) [inline keypress/base.js:9]: why the $'s infront of your variable names?

File: keypress/base.js [hunk 3]
```diff
@@ -2,4 +2,48 @@ console.log("Sanity Check: JS is working!");
...
+      $endTime = Date.now();
+        console.log ("endTime is now:" + $endTime);
+      $keyClick = false;
+        console.log ("keyClick is " + $keyClick);
+      $totalTime = ($endTime - $startTime); // computes duration between hits
+      var $convertedTime = milliToTime($totalTime);
+      $('#total-time').text($convertedTime);
+    }
+  });
+
```
Comment (Nick-Brennan) [inline keypress/base.js:29]: Great use of 'helper functions'

**Pull request closed without merging**